pages/definitions/index.tsx

```diff
@@ -17,17 +14,19 @@ const AllDefinitionsPage: NextPageWithLayout<{
       canonical="https://annuaire-entreprises.data.gouv.fr/definitions"
     />
     <StructuredDataFAQ
-      data={definitions.map(({ title, body }) => [
-        title,
-        renderToStaticMarkup(<ReactMarkdown>{body}</ReactMarkdown>),
-      ])}
+      data={definitions.map(({ title, body }) => [title, body.html])}
```

Comment:
We should use parseMarkdownSync here

But I dont know it it makes sense to still use `StructuredDataFAQ` as these pages looks increasingly like content pages and it is too long.